Component/team amanda + footer

amandaalexandre opened the pull request:
I added Linkedin links to the footer section so our credits can persist through navigation.

amandaalexandre committed:
feature: team with placeholder imgs

amandaalexandre committed:
feat: footer

mentalcaries commented on frontend/src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,31 @@
+            <p><a href="https://www.linkedin.com/in/amandavieiradev/">Amanda Vieira</a></p>
+            <p><a href="https://www.linkedin.com/in/devin-jaggernauth/">Devin Jaggernauth</a></p>
+            <p><a href="https://www.linkedin.com/in/henryque-lucena/">Henryque Lucena</a></p>
+            <p><a href="#">Muadd Ahmed</a></p>
```
Muuad Ahmed
https://www.linkedin.com/in/muaad-ahmed-a84a58157/

mentalcaries commented on frontend/src/components/Footer/Footer.tsx:
```diff
@@ -0,0 +1,31 @@
+        <div className='footer__team'>
+            <h2>Team</h2>
+            <br/>
+            <p><a href="https://www.linkedin.com/in/amandavieiradev/">Amanda Vieira</a></p>
```
Probably better to set these to open in a new tab
` <a href="https://google.com" target="_blank" rel="noopener noreferrer"></a>`

mentalcaries commented on frontend/src/components/Team/Team.tsx:
```diff
@@ -0,0 +1,62 @@
+        <h2>Meet our team</h2>
+        <br/>
+        <br/>
+        <div className='team__grid'>
```
You can consider making this a reusable component to avoid code duplication.
Not sure if `<br/>` is the best tag for semantic purposes. I know there were some issues with accessibility

mentalcaries reviewed (approved):
Approving for now so you can merge when ready

amandaalexandre committed:
fix: added Muadd's Linkedin

amandaalexandre committed:
fix: teams links open in another window

amandaalexandre commented:
- Added Muadd's link.
- Set link targets to different window as you requested
- Changed the bottom margin of the h2 element in that particular sectio
- I'll refactor the team code whenever we get our avatars or photos. I'll put them in a JSON and get the data from there. I'll add an issue to remind me to do it in the future.

amandaalexandre committed:
fix: h2 bottom margin
